packages/trader/src/App/Components/Elements/PositionsDrawer/__tests__/position-modal-card.spec.tsx

```diff
@@ -0,0 +1,178 @@
+        default_mock_props.contract_info.contract_type = TURBOS.LONG;
+        render(mockPositionsModalCard(mockStore(default_mock_store), default_mock_props));
+
+        expect(screen.queryByText(PositionsCardLoader)).not.toBeInTheDocument();
+        expect(screen.getByText(SymbolDisplayName)).toBeInTheDocument();
+        expect(screen.getByText('Long')).toBeInTheDocument();
+        expect(screen.getByText(/USD/i)).toBeInTheDocument();
+        expect(screen.getByText(/Buy price:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,671.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Contract value:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,517.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Entry spot:/i)).toBeInTheDocument();
+        expect(screen.getByText(/0.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Take profit:/i)).toBeInTheDocument();
+        expect(screen.getByText(/-/i)).toBeInTheDocument();
+        expect(screen.getByText(/Barrier:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,650/i)).toBeInTheDocument();
+    });
+    it('should render contract card for Rise/Fall', () => {
+        default_mock_props.contract_info.contract_type = 'rise_fall';
+        render(mockPositionsModalCard(mockStore(default_mock_store), default_mock_props));
+
+        expect(screen.queryByText(PositionsCardLoader)).not.toBeInTheDocument();
+        expect(screen.getByText(SymbolDisplayName)).toBeInTheDocument();
+        expect(screen.getByText(/USD/i)).toBeInTheDocument();
+        expect(screen.getByText(/Potential profit\/loss:/i)).toBeInTheDocument();
+        expect(screen.getByText(/Indicative price:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,517.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Buy price:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,671.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Payout limit:/i)).toBeInTheDocument();
+    });
+    it('should render the same contract card for Touch/No Touch as for Rise/Fall', () => {
+        default_mock_props.contract_info.contract_type = 'touch';
+        render(mockPositionsModalCard(mockStore(default_mock_store), default_mock_props));
+
+        expect(screen.queryByText(PositionsCardLoader)).not.toBeInTheDocument();
+        expect(screen.getByText(SymbolDisplayName)).toBeInTheDocument();
+        expect(screen.getByText(/USD/i)).toBeInTheDocument();
+        expect(screen.getByText(/Potential profit\/loss:/i)).toBeInTheDocument();
+        expect(screen.getByText(/Indicative price:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,517.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Buy price:/i)).toBeInTheDocument();
+        expect(screen.getByText(/2,671.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/Payout limit:/i)).toBeInTheDocument();
+    });
+});
```

Comment:
@kate-deriv since we don't change anything for Multipliers trade type, can we add a sanity test case for it, please? :pray::skin-tone-2:

Reply:
Sure